app/views/providers/allocations/index.html.erb

```diff
@@ -42,7 +46,7 @@
 
   <% if @allocations_view.initial_allocation_statuses.present? %>
     <div class="govuk-grid-column-full">
-      <h3 class="govuk-heading-m">New PE courses 2020/21</h3>
+      <h3 class="govuk-heading-l">New PE courses 2020/21</h3>
```

Comment:
Can we please keep this one `govuk-heading-m` as per the prototype?

Reply:
👍 do you have a link to prototype which shows the two tables on the page? I was trying to find it so that I could make sure it was all the same.

Reply:
If you go throughn the flow you get to: https://allocations-alpha.herokuapp.com/pe-allocations-new-courses

NOTE: This page looks ot have outdated copy - please ignore 😨